packages/charts/src/components/ColumnChartWithTrend/ColumnChartWithTrend.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  /*ChartContainer styles*/
+  font-size: var(--sapFontSmallSize);
+  color: var(--sapTextColor);
+  font-family: var(--sapFontFamily);
+  width: 100%;
+  height: 400px;
```

Comment:
Always 400px?

Reply:
Yup, that's the default for our other charts (implementing the `ChartContainer`) as well. We decided to keep a fixed height there, as otherwise the chart won't show up if the container doesn't implement a fixed height.